dotnet/src/SemanticKernel.Abstractions/Kernel.cs

```diff
@@ -156,6 +159,15 @@ public sealed class Kernel
         Interlocked.CompareExchange(ref this._autoFunctionInvocationFilters, [], null) ??
         this._autoFunctionInvocationFilters;
 
+    /// <summary>
+    /// Gets the collection of HTTP request message filters available through the kernel.
+    /// </summary>
+    [Experimental("SKEXP0001")]
+    public IList<IFunctionRequestMessageFilter> FunctionRequestMessageFilters =>
+        this._functionRequestMessageFilters ??
+        Interlocked.CompareExchange(ref this._functionRequestMessageFilters, [], null) ??
+        this._functionRequestMessageFilters;
```

Comment:
Presumably you also wanted to update AddFilters?

Reply:
That change has been removed